.github/workflows/cnf-test-partner-image.yaml

```diff
@@ -40,7 +40,7 @@ jobs:
           swap-storage: true
 
       - name: 'Build the `cnf-test-partner` image'
-        run: docker build  -t $IMAGE_NAME:$IMAGE_TAG -t $IMAGE_NAME:$SPECIFIC_IMAGE_TAG . --no-cache -f $IMAGE_CONTAINER_FILE_PATH
+        run: docker build --pull -t $IMAGE_NAME:$IMAGE_TAG -t $IMAGE_NAME:$SPECIFIC_IMAGE_TAG . --no-cache -f $IMAGE_CONTAINER_FILE_PATH
```

Comment:
The images used in the `Dockerfile` are versioned, do we need this pull?

Reply:
Good point, this is only really good for `latest`.  I will close.